docs/ansible/roles/debops.resources/defaults-detailed.rst

```diff
@@ -52,11 +52,18 @@ located in the :file:`ansible/resources/` directory (or wherever the
            └── hostname2/
 
 The ``with_filetree`` Ansible lookup plugin will look for resources to manage
-in specific hostname directory, then a specific group name directory defined by
-the :envvar:`resources__group_name` variable, then in the :file:`by-group/all/`
-directory. The resource found first in this order wins and no further checks
+in specific hostname directory, then of all the groups the current host is in
+(based on the content of the variable `group_names`), then in the :file:`by-group/all/` directory.
+The resource found first in this order wins and no further checks
 are performed; this means that you can put a file in the :file:`by-group/all/`
 directory and then override it using a host-specific directory.
+The groups directories are read in the order dictated by Ansible during inventory parsing.
+
+See `Ansible - Playbooks Varibles<https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables>`_
+to learn about the variable `group_names`.
+See `Ansible - Working with Inventory<https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#how-variables-are-merged>`_
```

Comment:
The references here seems to be misspelled and probably will result in errors during documentation generation. You could switch to using anonymous references for external links, for example:
```rst
See `Ansible - Playbooks Variables`__ to learn about the ``group_names`` variable, and `Ansible - Working with Inventory`__ for more information...

.. __: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables
.. __: https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#how-variables-are-merged
```